server/db/models/address.js

```diff
@@ -0,0 +1,37 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Address = db.define('address', {
+  streetAddress: {
+    type: Sequelize.TEXT,
+    allowNull: false,
+    validate: {
+      notEmpty: true
```

Comment:
@tungsteno22  Do we need both 'allowNull: false' AND 'notEmpty: true' validation ? Fine either way but not sure it's necessary or how it fits with best-practices